src/test/java/io/vertx/core/parsetools/FakeStream.java

```diff
@@ -24,6 +24,8 @@
   private Handler<Buffer> eventHandler;
   private Handler<Void> endHandler;
   private Handler<Throwable> exceptionHandler;
+  int pauseCount;
```

Comment:
make it private and add a pauseCount() accessor

Reply:
done